backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ListenerApiImpl.scala

```diff
@@ -149,11 +149,11 @@ class ListenerApiImpl extends ListenerApi {
     val libPath = conf.get(GlutenConfig.GLUTEN_LIB_PATH, StringUtils.EMPTY)
     if (StringUtils.isNotBlank(libPath)) { // Path based load. Ignore all other loadees.
       JniLibLoader.loadFromPath(libPath, false)
-      return
+    } else {
```

Comment:
It looks a bug. We usually don't set GLUTEN_LIB_PATH in config, so the issue is not exposed.
Maybe, this config is useless now for velox backend, but may keep it here for future use possibly. cc @zhztheplayer

Reply:
@wForget So are you actually using `GlutenConfig.GLUTEN_LIB_PATH`?

Anyway the change still looks reasonable even if no one is using the option.

Reply:
> @wForget So are you actually using `GlutenConfig.GLUTEN_LIB_PATH`?

no, just noticed while reading the code